src/SprykerMiddleware/Zed/Process/Business/ProcessFacadeInterface.php

```diff
@@ -13,4 +13,28 @@
      * @return array
      */
     public function map(array $payload, MapInterface $map);
+
+    /**
+     * @param array $payload
+     * @param array $dictionary
+     *
+     * @return array
+     */
+    public function translate(array $payload, array $dictionary);
+
+    /**
+     * @param array $payload
+     * @param string $destination
+     *
+     * @return array
+     */
+    public function writeSerialized(array $payload, string $destination);
```

Comment:
I think the process shouldn't know about concrete details, like writers type. Should be like writePayload and 
 a writer has to be selected based on a configuration.